common/LongPathHandler.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"os"
+	"strings"
+)
+
+func ToLongPath(short string) string {
+	if os.PathSeparator == '\\' {
+		if strings.HasPrefix(short, `\\?\`) {
+			return strings.Replace(short, `/`, `\`, -1)
+		} else if strings.HasPrefix(short, `\\`) {
+			return strings.Replace(`\\?\UNC`+short[1:], `/`, `\`, -1)
+		} else {
+			return strings.Replace(`\\?\`+short, `/`, `\`, -1)
+		}
+	}
+
+	return short
+}
+
+func ToShortPath(long string) string {
+	if os.PathSeparator == '\\' {
+		if strings.HasPrefix(long, `\\?\UNC`) {
+			return strings.Replace(`\`+long[7:], `\`, `/`, -1)
+		} else if strings.HasPrefix(long, `\\?\`) {
+			return strings.Replace(long[4:], `\`, `/`, -1)
+		}
+	}
```

Comment:
This is changing not just the prefx, but also the direction the /, right?  I find that a bit confusing.  It might be just a namming thing.  E.g. maybe call ToLongPath something like "ToWindowsLongPath" and toShortPath something like "ToAzCopyShortPath" or "ToUnixShortPath" or soemthing. That might make it more understandable. Or alternatively, separate the concepts of long/short (i.e;. prefixed/unprefixed) vs / and \.   But i think the naming change will be the easier fix.

Reply:
I brain farted on both of these entirely. I don't need to fix the direction of the `/`, and I also need to return a `\` to the UNC path.